best-practices.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width" />
+        <title>Best practices and recommended tools</title>
+        <link rel="stylesheet" href="css/wgio.css" />
+        <link rel="icon" type="image/x-icon" href="//labs.w3.org/favicon.ico" />
+    </head>
+    
+    <body>
+      
+        <header>
+            <h1>Best practices and recommended tools</h1>
+        </header>
+        
+        <nav>
+            <a href="/">Home</a>
```

Comment:
These should be inside an `<ul>` -> `<li>` to be valid.

Reply:
Is it a bug that [the validator doesn't complain](https://validator.w3.org/nu/?doc=https%3A%2F%2Frawgit.com%2Fw3c%2Fw3c.github.io%2Ftripu%2Fbest-practices%2Fbest-practices.html)?

Reply:
[Looking at it](https://www.w3.org/TR/html5/) again:

> *&ldquo;A `nav` element doesn’t have to contain a list, it can contain other kinds of content as well. In this navigation block, links are provided in prose.&rdquo;*

&mdash; [Example 11](https://www.w3.org/TR/html5/sections.html#example-84241bb3)

And:

> *&ldquo;In cases where the content of a `nav` element represents a list of items, use list markup to aid understanding and navigation.&rdquo;*

&mdash; [Note](https://www.w3.org/TR/html5/sections.html#ref-for-elementdef-nav%E2%91%A6)

So I understand that this short `nav` with three links is valid as it is.

Reply:
TIL :)